frontend/src/routes/signup/+page.svelte

```diff
@@ -0,0 +1,8 @@
+<script>
+  import Signup from '../../lib/components/Signup.svelte';
```

Comment:
The problem here is `signup.svelte` is named with lower case, but referenced as `Signup.svelte` here. All the other components are named in title case so for consistence I would rename `signup.svelte` to `Signup.svelte`

This is the fix for how things are now.

```suggestion
  import Signup from '$lib/components/signup.svelte';
```